doc/README.md

```diff
@@ -174,23 +178,63 @@ pprof can generate graphical reports on the DOT format, and convert them to
 multiple formats using the graphviz package.
 
 These reports represent the location hierarchy as a graph, with a report entry
-represented as a node. Solid edges represent a direct connection between
-entries, while dotted edges represent a connection where some intermediate nodes
-have been removed. Nodes are removed using heuristics to limit the size of
+represented as a node. Nodes are removed using heuristics to limit the size of
 the graph, controlled by the *nodecount* option.
 
-The size of each node represents the flat weight of the node, and the width of
-each edge represents the cumulative weight of all samples going through
-it. Nodes are colored according to their cumulative weight, highlighting the
-paths with the highest cum weight.
-
-* **-dot:** Generates a report in .dot format. All other formats are generated from
-  this one.
+* **-dot:** Generates a report in .dot format. All other formats are generated
+  from this one.
 * **-svg:** Generates a report in SVG format.
 * **-web:** Generates a report in SVG format on a temp file, and starts a web
   browser to view it.
-* **-png, -jpg, -gif, -pdf:** Generates a report in these formats,
+* **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
+
+### Interpreting the Callgraph
+
+* **Flat Value**: the value of a location itself. This is indicated by the font 
+  size.
+    *   Nodes with larger font have larger flat values.
+    *   Nodes with smaller font have smaller flat values.
+
+* **Cum Value**: the value of a location plus all of its descendants. This is
+  indicated by the nodes' color.
```

Comment:
Nit: "location" is singular, so "nodes'" should be singular (node's)

Reply:
Done.